lib/MDButtonWithSpinner.js

```diff
@@ -45,9 +45,7 @@ var MDButtonWithSpinner = function MDButtonWithSpinner(props) {
     content = _react2.default.createElement(
       _mdbutton2.default,
       buttonProps,
-      ' ',
-      children,
-      ' '
+      children
```

Comment:
rigth~.but we don't have to lint the compiled code, src code is in src/**.js
and I have fixed bugs in test #5 . This PR will be closed 2 hours later